.mvn/wrapper/maven-wrapper.properties

```diff
@@ -14,5 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.3/apache-maven-3.9.3-bin.zip
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
+distributionSha256Sum=7822eb593d29558d8edf87845a2c47e36e2a89d17a84cd2390824633214ed423
```

Comment:
Here and just here, I would like us to keep 3.9.3 as it's the only way to be in sync with mvnd (I'm experimenting with the Gradle build cache and I'm trying to use the exact same version). Can you revert this one change, please?

Reply:
@gsmet , Yes of course I'll will update this pull request sooner

Reply:
@gsmet  I already updated the pull request and rebased on main branch